Tests/FluentMySQLTests/FluentMySQLTests.swift

```diff
@@ -205,6 +205,56 @@ class FluentMySQLTests: XCTestCase {
         test = try test.save(on: conn).wait()
     }
 
+    func testReferences() throws {
+        database.enableLogging(using: .print)
+        let conn = try benchmarker.pool.requestConnection().wait()
+        defer { benchmarker.pool.releaseConnection(conn) }
+
+        // Prep tables
+        defer { try? Child.revert(on: conn).wait()
+                try? Parent.revert(on: conn).wait() }
+        try Parent.prepare(on: conn).wait()
+        try Child.prepare(on: conn).wait()
+        // Save Pet
+        let parent = Parent(id: 64, name: "Snuffles")
+        let savedParent = try parent.create(on: conn).wait()
```

Comment:
Shouldn't `parent.id == savedParent.id` ? The value is stored correctly on MySQL (64) but `savedParent.id` is `0`

Reply:
you should probably do `var parent` and then update the parent property. Since it's a struct, the `parent` variables id cannot be set asynchronously. That's why `savedParent` is returned